Convert jmeter to service

cthayes opened the pull request:
This PR refactors a bunch of the test to start the jmeter test as service running as a new user instead of root.
Also changed the AWS account this runs against.

BLUEBUTTON-619

cthayes committed:
Convert jmeter to service

Also changed the AWS account this runs against.

BLUEBUTTON-619

karlmdavis commented on ansible/templates/bluebutton-jmeter.service.j2:
```diff
@@ -0,0 +1,13 @@
+WorkingDirectory={{ remote_test_dir }}
+ExecStart={{ remote_test_dir }}/bluebutton-jmeter.sh
+User={{ remote_jmeter_user }}
+RemainAfterExit=true
```
What's this for?

karlmdavis commented on ansible/templates/bluebutton-jmeter.sh.j2:
```diff
@@ -0,0 +1,8 @@
+-Dserver.rmi.localport={{ jmeter_server_rmi_local_port }} \
+-Dserver_port={{ jmeter_server_port }} \
+-j {{ remote_test_dir }}/log-jmeter.txt \
+&> {{ remote_test_dir }}/log-jmeter-console.txt &
```
Any idea why you have to run this in the background?

karlmdavis reviewed (approved):
In general? :100: 

Would like to see if you can figure out the "why does this have to be backgrounded" thing, just because I think it'd be educational for the bother of us, but feel free to merge either way.

cthayes committed:
Merge branch 'master' into BLUEBUTTON-619

cthayes committed:
Use async ansible task with jmeter in foreground

- Switched JMeter to run int he foreground instead of background

- Used ansible async task to kick it off

BLUEBUTTON-619

cthayes commented:
Turns out we didn't need to run JMeter in the background. Just needed to make the ansible task an async one.
